lib/active_admin/views/pages/show.rb

```diff
@@ -25,7 +25,11 @@ def main_content
         end
 
         def attributes_table(*args, &block)
+          opts = args.extract_options!
           table_title = ActiveAdmin::Localizers.resource(active_admin_config).t(:details)
+          if given_title = opts[:title] || given_title = opts[:header]
```

Comment:
No good reason, I guess. I'd have been inclined to use only `:title`, but I think I saw one or two people mention `:header`, so I figured I'd have both. Do you have a preference for one or the other?